src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/templates/InternalAdaGenerateHearingBundleTemplate.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.templates;
```

Comment:
Can we please use the letter package for all the notification related letter generation?